lib/Controller/LoginController.php

```diff
@@ -344,6 +353,10 @@ public function code($state = '', $code = '', $scope = '', $error = '', $error_d
 		$this->userSession->createSessionToken($this->request, $user->getUID(), $user->getUID());
 		$this->userSession->createRememberMeToken($user);
 
+		// for backchannel logout
+		$this->config->setAppValue(Application::APP_ID, 'sid-' . $idTokenPayload->sid, $this->session->getId());
+		$this->session->set(self::CURRENT_SID, $idTokenPayload->sid);
```

Comment:
So we actually need to store this in the session or could we just get the current authtoken somehow? Session might not be available if the remember me cookie is used, iirc.